src/tooling/debugging/index.md

```diff
@@ -4,12 +4,36 @@ Debugging Rust applications is also possible using different tools that will be
 
 Refer to the table below to see which chip is supported in every debugging method:
 
-|              | **probe-rs** | **OpenOCD** | **VS Code** |
-| :----------: | :----------: | :---------: | :---------: |
-|  **ESP32**   |      ❌       |      ✅      |      ✅      |
-| **ESP32-C2** |      ✅       |      ✅      |      ✅      |
-| **ESP32-C3** |      ✅       |      ✅      |      ✅      |
-| **ESP32-C6** |      ✅       |      ✅      |      ✅      |
-| **ESP32-H2** |      ✅       |      ✅      |      ✅      |
-| **ESP32-S2** |      ❌       |      ✅      |      ✅      |
-| **ESP32-S3** |      ❌       |      ✅      |      ✅      |
+|              | **probe-rs** | **OpenOCD** |
+| :----------: | :----------: | :---------: |
+|  **ESP32**   |      ❌       |      ✅      |
+| **ESP32-C2** |      ✅       |      ✅      |
+| **ESP32-C3** |      ✅       |      ✅      |
+| **ESP32-C6** |      ✅       |      ✅      |
+| **ESP32-H2** |      ✅       |      ✅      |
```

Comment:
ESP32-C2 and ESP32-H2 should be in progress, hopefully the support will be added soon